examples/fabric/v4/portConnectivity/port2portself/terraform.tfvars

```diff
@@ -1,5 +1,6 @@
-equinix_client_id      = "MyEquinixClientId"
-equinix_client_secret  = "MyEquinixSecret"
+/* Client details for opsuser100 in uat with the UAT1 App in Developer Portal */
+equinix_client_id      = "V9JsDiT9AROAAZfrShFg9SeTOimUDRLBmjvwVVNErbPZ4aAE"
+equinix_client_secret  = "wezmRx5jklOuxm1kKJSeSdTxLZVatNbusz30PhTje8nr5XvNj05boJ2sr6l3ctpZ"
```

Comment:
@thogarty Please do not commit this. You would trigger an info sec email chain and might need to work with Gautham to get this secret invalidated immediately.